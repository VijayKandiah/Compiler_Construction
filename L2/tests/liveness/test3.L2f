(:myF
  1 0

  %myVar1 <- 0

  %myVar1 += rdi

  rax <- %myVar1

  return
)
