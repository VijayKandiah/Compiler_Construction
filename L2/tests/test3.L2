(:go
  (:go
    0 0

    %v1 @ %v2 %v3 4
    rax <- %v1

    return
  )
)
