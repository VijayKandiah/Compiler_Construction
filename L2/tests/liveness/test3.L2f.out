(
(in
(r12 r13 r14 r15 rbp rbx rdi)
(myVar1 r12 r13 r14 r15 rbp rbx rdi)
(myVar1 r12 r13 r14 r15 rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
)

(out
(myVar1 r12 r13 r14 r15 rbp rbx rdi)
(myVar1 r12 r13 r14 r15 rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
()
)

)
